pkg/networkservice/chains/nsmgr/server_heal_test.go

```diff
@@ -368,7 +374,7 @@ func testNSMGRHealNSMgr(t *testing.T, nodeNum int, customConfig []*sandbox.NodeC
 	e, err := nsc.Close(ctx, conn)
 	require.NoError(t, err)
 	require.NotNil(t, e)
-	require.Equal(t, int32(2), atomic.LoadInt32(&counter.Requests))
```

Comment:
@Bolodya1997 Why this is changed?

Reply:
There was a bug in code accidentally made by Artem. Initially there was a test without refresh request, after it was skipped due to the https://github.com/networkservicemesh/sdk/issues/770 and after there were added refresh request (but the test has already been skipped and so this count was not updated).